web/src/css/flowview.less

```diff
@@ -6,4 +6,28 @@
         max-width: 100%;
         max-height: 100%;
     }
-}
\ No newline at end of file
+}
+
+.edit-flow  {
+    position: absolute;
+    right: 25px;
+    top: 140px;
```

Comment:
If you wrap `.edit-flow` in a positioned div, we can put in useful values here. These `140px` are dependend on the height of the header etc, that's very fragile.